### ISSUE TITLE
Ckeditor - Provide method for allowing clients to insert automatically responsive tables

### ISSUE THREAD

**Issue by mmafe** [ccswbs/hjckrrh#591]
Currently, it looks like the table-responsive class is being added to the actual table inserted by ckeditor.

It turns out, this should be added to a container (div) surrounding the table.

See http://getbootstrap.com/css/#tables-responsive

Code should look like:
```
<div class="table-responsive">
  <table class="table">
    ...
  </table>
</div>
```

**Comment by ushenoyuog** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Ckeditor - Provide method for allowing clients to insert automatically responsive tables](https://trello.com/c/nJqmLT0d/222-ckeditor-provide-method-for-allowing-clients-to-insert-automatically-responsive-tables)

**Comment by ushenoyuog** [comment]
Closing issue as it seems to have been merged and accepted.
